microsoft/testsuites/adhoc/tool_install.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, List, Union
+
+from lisa import (
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    tools,
+)
+from lisa.executable import Tool
+from lisa.util import LisaException, SkippedException
+
+
+@TestSuiteMetadata(
+    area="adhoc",
+    category="functional",
+    description="""
+    This test suite includes adhoc tests
+    """,
+)
+class Adhoc(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will install the tools
+        passed as parameter 'install_tools'
+        """,
+        priority=5,
+    )
+    def tool_install(
+        self, log: Logger, node: RemoteNode, variables: Dict[str, Any]
+    ) -> None:
+        tool_name_parameter = "install_tools"
+        tool_names: List[str] = variables.get(tool_name_parameter, [])
+        if not tool_names:
+            raise SkippedException(f"{tool_name_parameter} is empty")
```

Comment:
Give more context like, this test case is used to deploy tools in an environment, it doesn't use for validating.